config/core/configmaps/gc.yaml

```diff
@@ -60,27 +60,52 @@ data:
     # V2 Garbage Collector Settings
     # ---------------------------------------
     #
-    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # These settings are enabled via the "responsive-revision-gc" feature flag.
     # ALPHA NOTE: This feature is still experimental and under active development.
+    #
+    # Stale
+    #   * Revisions are considered stale if they were created after more than
+    #      "retain-since-create-time" or last referenced by a route more than
+    #      "retain-since-last-active-time" ago.
```

Comment:
Is it `or` here or is it `and`? With `or` if I put since creation time to be, say 1m and keep since last active at 15h, then in a minute after losing route this is a candidate for collection.
Also is there any restrictions on the relative values of either? E.g. does one has to be necessarily smaller than the other?

Reply:
you're right, its an 'and' as written.